### PR TITLE
Task 1: Initialized React Project; Created simple React components

### PR DESCRIPTION
**Pull request by ykukharskyi**
… different ways

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# React Mentoring Program
+
+#### Task 1: Setting up the environment
+
+```$bash
+$ npm init
+$ npm install express --save
```
Comment (torokhkun) [inline README.md]: You have already had "express" as a dependency in your project.json file so you can type `npm i` instead.
And may be the good point to add `npm start` command for running application.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+.idea/
+*.iml
```
Comment (torokhkun) [inline .gitignore]: For what purposes we ignore `iml` files?
Even more, how `iml` files can arise in our project?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# React Mentoring Program
+
+#### Task 1: Setting up the environment
+
+```$bash
+$ npm init
```
Comment (torokhkun) [inline README.md]: You have already had project.json file in your project so you don't need retype this command.

File: .gitignore (hunk 2 above)
Comment (ykukharskyi) [reply .gitignore]: `iml` files can be generated by IntelliJ IDEA. I'm going to use this IDE for experimenting, so I need `iml` file mask on the `.gitignore` for preventing pushing such files to the repository.

File: README.md (hunk 1 above)
Comment (ykukharskyi) [reply README.md]: It was added just for history. Will remove it.

File: README.md (hunk 3 above)
Comment (ykukharskyi) [reply README.md]: It was added just for history. Will remove it.

**Pull request merged**